drag_and_drop_v2/templates/html/drag_and_drop.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
-<section class="themed-xblock xblock--drag-and-drop">
-    <i class="fa fa-spin fa-spinner initial-load-spinner"></i>{% trans "Loading drag and drop problem." %}
-</section>
+<div class="themed-xblock xblock--drag-and-drop">
+    <span class="fa fa-spin fa-spinner initial-load-spinner"></span>{% trans "Loading drag and drop problem." %}
```

Comment:
The icons need to be marked with aria-hidden. See the HTML markup at http://ux.edx.org/design_elements/icons/.